actionview/lib/action_view/template/error.rb

```diff
@@ -59,13 +59,16 @@ class Template
     class Error < ActionViewError #:nodoc:
       SOURCE_CODE_RADIUS = 3
 
-      attr_reader :original_exception
+      def initialize(template, message, backtrace)
+        super(message)
+        set_backtrace(backtrace)
```

Comment:
Yes, we can `delegate :backtrace, :to_s, to: :cause`. We don't have to `delegate :message, to: :cause` since what `#message` does is to [just call `#to_s` internally](http://www.ruby-doc.org/core-2.2.0/Exception.html#method-i-message).